accounting/transferwise/admin.py

```diff
@@ -22,20 +22,35 @@
 """
 
 from django.contrib import admin
+from django.utils.html import format_html
+from rangefilter.filter import DateRangeFilter
 
 from accounting.common.admin import UuidModelAdmin
 from accounting.transferwise import models
 
 
+# pylint: disable=no-self-use
 @admin.register(models.TransferWiseBulkPayment)
 class TransferWiseBulkPaymentAdmin(UuidModelAdmin):
     """ Admin configuration for the `TransferWiseBulkPayment` model. """
-    list_display = ('date', 'csv_path',)
-    readonly_fields = ('csv_path',)
+
+    list_display = ('__str__', 'csv',)
```

Comment:
@UmanShahzad Similar note here about renaming the `csv` method to `csv_link`.

Reply:
:+1: